docs/content/docs/features/lookup-joins.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Lookup Joins"
```

Comment:
I prefer to put this into SQL API, and add a `Flink-Only`.

Reply:
Lookup join seems more like a feature to me. The "SQL API" section should only contain basic and common usage of Table Store, while more advanced usage should be moved to the "Features" section.

Reply:
Lookup join seems more like a feature to me. The "SQL API" section should only contain basic and common usage of Table Store, while more advanced usage should be moved to the "Features" section.